src/klein/test/util.py

```diff
@@ -2,28 +2,30 @@
 Shared tools for Klein's test suite.
 """
 
+from abc import ABC, abstractmethod
 
-class EqualityTestsMixin(object):
+
+class EqualityTestsMixin(ABC):
```

Comment:
Use an ABC and mark methods abstract instead of raising when called; this catches code errors immediately and removes untested code.